coalib/coala.py

```diff
@@ -74,11 +98,8 @@ def main(debug=False):
         elif args.show_capabilities:
             from coalib.collecting.Collectors import (
                 filter_capabilities_by_languages)
-            from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
+            local_bears, global_bears = FilterHelper.get_filtered_bears(
+                'language', args.filter_by_language)
```

Comment:
Analyzing it properly, it should ideally be `args.show_capabilities` since languages are given as the arguments of option `show_capabilities` as per tests.
So, `coala -p java` works fine. No need of `coala -p -l java`.